__tests__/review.test.js

```diff
@@ -2,14 +2,21 @@ import { render, screen, cleanup } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import App from '../client/src/app.jsx';
 import React from 'react';
-import reviewData from '../reviewSampleData.json';
-import { configure, shallow } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import { configure, shallow, mount } from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
Is this adapter just to stay current with the react version we are using?

Reply:
yes, since we are using the "react": "^17.0.2", the react-16 doesn't support some functions for version 17, like mount..